.github/workflows/build_and_test.yml

```diff
@@ -128,8 +128,6 @@ jobs:
 
     - name: Build Storybook to sanity check components
       run: npm run build-storybook ; rm -rf ./storybook-static
-      env:
-        NODE_OPTIONS: --max_old_space_size=8192
```

Comment:
We need all this memory to build the storybook? 4096 isn't enough?

Reply:
We need it (at least while we don't fix the typings issues I've cited), but this heap size is now set on the npm script. Take a look at `"storybook"` and `"build-storybook"` in `package.json`.

Reply:
I mean, I had a memory issue to run the storybook I was thinking if it's necessary 8192 because I solve using 4096, doesn't affect the review, it's just for curiosity :P

Reply:
Btw, we still have the issue with cookie and cookie-parser when trying to launch storybook :disappointed: